src/test/java/org/sonarsource/sonarlint/ls/mediumtests/ConnectedModeMediumTests.java

```diff
@@ -166,6 +167,11 @@ protected void verifyConfigurationChangeOnClient() {
     }
   }
 
+  @AfterAll
+  public static void cleanUp() {
+    folder1BaseDir.toFile().delete();
```

Comment:
```suggestion
    org.apache.commons.io.FileUtils.deleteQuietly(folder1BaseDir.toFile());
```

IMHO, using `java.io.File.delete()` still has a risk of failing with an `IOException`.

With `FileUtils.deleteQuietly(File)`, any exception thrown while recursively deleting the folder will be caught and avoid failing the tests.